core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.js

```diff
@@ -94,6 +94,12 @@ oppia.directive('audioTranslationBar', [
           $scope.audioTimerIsShown = true;
           $scope.audioIsCurrentlyBeingSaved = false;
 
+          document.body.onkeyup = function(e) {
+            if (e.keyCode === 32) {
```

Comment:
I used space key as it is significantly wider and can be easily identified. I will discuss this with @DubeySandeep and confirm if I have to change it or not.
Thank you

Reply:
Sure. :)
I think @DubeySandeep will have the same opinion on this.
If the browser is not maximized, i.e, if the browser size is small, then pressing space will completely disturb the exploration author.(Try pressing space button with a small browser size).

Also, we need to introduce a small note telling the author that he/she can start/stop the recorder using the shortcut key(otherwise there is no way the authors would come to know about the new functionality).

Something like this:
![image](https://user-images.githubusercontent.com/32777813/52969874-eb4a2180-33d7-11e9-8969-663792a550e4.png)

And this note should only be displayed when the shortcut is expected to work, i.e., it should only be shown when:
 1. Audio translation is not present in the current sub-tab.
 2. Audio translation needs update.

Reply:
@suicide11, I'm not sure about what to have as a "shortcut key" for recording audio. I would suggest you to find your answers through these options:
- Ask the end user (translators), what they would preffer.
- Check for the shortcut used in the existing popular recording apps (Like audacity)
- I think space bar makes the most noise so do check that we don't record the clicking noise at the begging and end of the recording.

I think having a result to these options will be a better way to find the answer.

Reply:
Also, I liked the idea of having a tolltip there. We can have it as a ProTip maybe? (Found this on github)
![image](https://user-images.githubusercontent.com/16653571/53181157-09549380-361d-11e9-832b-89c57b301b6c.png)

Reply:
Ok, I will make my next commit by tomorrow and will try to include all the above reviews.

Reply:
Done